Microsoft.Toolkit.Uwp.Notifications/Microsoft.Toolkit.Uwp.Notifications.csproj

```diff
@@ -20,19 +21,15 @@ Supports adaptive tiles and adaptive/interactive toasts for Windows 10. It is pa
   <ItemGroup>
     <None Include="Microsoft.Toolkit.Uwp.Notifications.targets" Pack="true" PackagePath="build\native" />
   </ItemGroup>
-
-  <ItemGroup>
-    <PackageReference Include="System.Xml.ReaderWriter" Version="4.3.0" />
-  </ItemGroup>
   
 
   <PropertyGroup Condition="'$(TargetFramework)' == 'native'">
     <OutputType>winmdobj</OutputType>
     <TargetPlatformIdentifier>UAP</TargetPlatformIdentifier>
     <NugetTargetMoniker>UAP,Version=v10.0</NugetTargetMoniker>
     <PackageTargetFallback>uap10.0</PackageTargetFallback>
-    <TargetPlatformVersion Condition="'$(TargetPlatformVersion)' == '' ">10.0.10240.0</TargetPlatformVersion>
-    <TargetPlatformMinVersion Condition="'$(TargetPlatformMinVersion)' == '' ">10.0.10240.0</TargetPlatformMinVersion>
+    <TargetPlatformVersion Condition="'$(TargetPlatformVersion)' == '' ">10.0.15063.0</TargetPlatformVersion>
```

Comment:
We need to keep the minimum for the notifications package to 10240 by request from the notification team.

Reply:
Does it need to be the just the min version or also the TPV? the TPV means that you must have the 10240 SDK installed to build, which a growing number of people won't have locally anymore.

Reply:
Okay, I've pushed an update that sets the TPMV to 10240 for the WinMD components. 

Couple other questions: does the TFM In the nuget need to be `windows` or can/should it be `uap10.0`?

Is there any update on getting higher min versions supported for JS/C++?

Reply:
Tested on C#/C++/JS/WPF, all good!

TFM in the nuget likely needs to be `windows` as it is. Otherwise I think some issues occur installing on C++ or something. Everything is set up as it is for a reason. If you want to make changes, you'll have to test that it all still works on UWP C#/C++/JS and non-UWP C# like WPF, servers, etc

Reply:
The `Windows` TFM is specifically for Windows 8. Windows 10 does not use that TFM. I don't think that's really the right answer, but it depends on how the project system sees it. @rrelyea, do you know what happens for WinJS?

For C++, it uses the `native` TFM.